src/Tizen.NUI/src/internal/WebView/WebHitTestResult.cs

```diff
@@ -226,6 +226,7 @@ public ImageView Image
                 ImageView image = new ImageView(url.ToString());
                 image.Size = new Size(pixelData.GetWidth(), pixelData.GetHeight());
```

Comment:
Do we have to set `Size` value manually?
@hinohie Do you know about it?

Reply:
(This comment is not related with this PR, so I'm going to merge this anyway)

Reply:
I checked to the DALi team. Getting `Size` property right after create ImageView can return invalid size so it's better to set it manually.

Reply:
More detail / If we don't set `Size` manually, `View.WidthSpecification` series didn't setup (default as `WrapContent`).  If we set `Size` manually, `View.WidthSpecificatoin` setup fixed so most calculation at `layout` will be more light.